Retroblur/color.ini

```diff
@@ -0,0 +1,215 @@
+; COLOR EXPLANATION
```

Comment:
There are several color schemes. You should provide a screenshot for each of them and structure the README as specified in the [contribution guidelines](https://github.com/morpheusthewhite/spicetify-themes/blob/master/CONTRIBUTING.md)